### PR TITLE
Remove 'address' field from 'person' model

### LINKED ISSUE #38: Rename seedu.address.model classes
Rename classes as:
- Person &rarr; Transaction
  - Name &rarr; Name
  - Phone &rarr; Amount
  - Tags &rarr; Category
  - Email &rarr; Date
  - Address &rarr; (deleted)

### PR DESCRIPTION
**Pull request by ianyong**
Resolves #38.

### CODE REVIEW

File: src/test/java/seedu/address/logic/parser/AddCommandParserTest.java [hunk 1]
```diff
@@ -49,36 +44,36 @@ public void parse_allFieldsPresent_success() {
...
         // multiple dates - last date accepted
         assertParseSuccess(parser, NAME_DESC_BOB + AMOUNT_DESC_BOB + DATE_DESC_AMY + DATE_DESC_BOB
-                + ADDRESS_DESC_BOB + CATEGORY_DESC_FRIEND, new AddCommand(expectedTransaction));
+                + CATEGORY_DESC_FRIEND, new AddCommand(expectedTransaction));
 
         // multiple addresses - last address accepted
-        assertParseSuccess(parser, NAME_DESC_BOB + AMOUNT_DESC_BOB + DATE_DESC_BOB + ADDRESS_DESC_AMY
-                + ADDRESS_DESC_BOB + CATEGORY_DESC_FRIEND, new AddCommand(expectedTransaction));
+        assertParseSuccess(parser, NAME_DESC_BOB + AMOUNT_DESC_BOB + DATE_DESC_BOB + CATEGORY_DESC_FRIEND,
+                new AddCommand(expectedTransaction));
```
Comment (zhaojj2209) [inline src/test/java/seedu/address/logic/parser/AddCommandParserTest.java:63]: This testcase can be deleted.

File: src/test/java/seedu/address/model/person/NameContainsKeywordsPredicateTest.java [hunk 2]
```diff
@@ -70,6 +70,6 @@ public void test_nameDoesNotContainKeywords_returnsFalse() {
         // Keywords match amount, date and address, but does not match name
```
Comment (zhaojj2209) [inline src/test/java/seedu/address/model/person/NameContainsKeywordsPredicateTest.java:70]:
```suggestion
        // Keywords match amount and date, but does not match name
```
Not sure if this testcase will still be used in the future, but we can change the description first.

**Review by zhaojj2209 — DISMISSED** [review]
LGTM!

**Review by siddarth2824 — COMMENTED** [review]
Remove AddressTest class since we do not have 'address' field in 'person' model anymore

**Review by siddarth2824 — APPROVED** [review]
LGTM!

**Review by zhaojj2209 — APPROVED** [review]
LGTM!

**Pull request merged**